client/sites-dashboard/components/wpcom-smp-commands.tsx

```diff
@@ -403,6 +404,15 @@ export const useCommandsArrayWpcom = ( {
 			},
 			icon: addNewSiteIcon,
 		},
+		{
```

Comment:
Also, not sure if we should display this as the default command when the user is outside of `/sites` in case we decided to add something as opposed to keeping only the search field

Reply:
- We will handle it https://github.com/Automattic/dotcom-forge/issues/4673